src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -4388,6 +4518,17 @@ def set_up_windows_profile(self, mc: ManagedCluster) -> ManagedCluster:
             mc.windows_profile = windows_profile
         return mc
 
+    def set_up_storage_profile(self, mc: ManagedCluster) -> ManagedCluster:
+        """Set up storage profile for the ManagedCluster object.
+
+        :return: the ManagedCluster object
+        """
+        self._ensure_mc(mc)
+
+        mc.storage_profile = self.context.get_storage_profile()
```

Comment:
[CI](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1804092&view=logs&j=4e4d232d-230a-55c9-0730-4c1a9c36b9e5&t=6d1bdb99-245b-5117-da18-01d8df6f72c0&l=1967) failed due to no such profile in API version `2020-11-01`. A simple workaround would be add a `hasattr` check here.

```suggestion
        if hasattr(self.models, "ManagedClusterStorageProfile"):
            mc.storage_profile = self.context.get_storage_profile()
```

Reply:
done.